mantidimaging/gui/windows/stack_visualiser/metadata_dialog.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import absolute_import, division, print_function
+
+from PyQt5 import Qt
+
+
+class MetadataDialog(Qt.QDialog):
```

Comment:
The pretty printing is good but when the dialog first opens for me I immediately have to resize it or the text wraps and messes up the pretty printing anyway. Should we try and set a slightly wider dialog when it first opens?

Reply:
Widened to 600px by default.

I was hoping Qt could be able to dynamically resize it (as you can get the displayed document width of a `QTextEdit`), however the results were always too narrow.

Reply:
I've always found Qt's sizing rules to be quite mysterious. I think the new default width is a good compromise. :+1: